src/exiv2.1

```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
 with the 'rename' action. Note: On Windows you may have to set the TZ
 environment variable for this option to work correctly.
 .TP
-.B \-f
-Do not prompt before overwriting existing files (force overwrite).
+.B \-f,\-F
+These options are used by commands 'rename' and 'extract' to determine file overwrite policy.  These options
+are usually combined with -v/--verbose to provide additional status output.
+.br
+.sp 1
+The options --force and --Force apply to the 'rename' command.  The 'extract' command treats --force and --Force as permission to overwrite.
+.br
+.sp 1
```

Comment:
Default behaviour is to prompt user. -> The default behaviour is to prompt the user.